src/Controllers/BatchProcessorController.ts

```diff
@@ -45,36 +60,53 @@ module StreamStats.Controllers {
         public selectedBatchProcessorTabName: string;
         public displayMessage: string;
         public isValid: boolean;
-        public AppVersion: string;  
+        public AppVersion: string;
         public submitBatchInfo: string;
+
+        // Regions
         public regionList: Object;
-        public flowStatsList: Array<any>;
         public selectedRegion: string;
+
+        // Flow Stats
+        public flowStatsList: Array<any>;
         public flowStatsAllChecked: boolean;

```

Comment:
I don't think this variable is being used.

Reply:
This variable is used.